docs/source/guides/contributor/chapters/runners.rst

```diff
@@ -26,23 +26,25 @@ There are a number of reasons for introducing a different architecture
 and implementation.  Some of them are related to limitations found in
 the current implementation, that were found to be too hard to remove
 without major breakage.  Also, missing features that are deemed
-important were fit better in a different architecture.
+important would be a better fit wihin a different architecture.
 
 For instance, these are the current limitations of the Avocado test
 runner:
 
 * Test execution limited to the same machine, given that the
-  communication between runner and test is a Python queue (the remote
-  runner plugins actually execute an Avocado Job remotely, with all
-  the overhead and complications that it brings)
-* Test execution is limited to a single test at a time (non-parallel)
+  communication between runner and test is a Python queue (the
+  "remote" runner plugins Avocado once had would actually execute an
+  Avocado Job remotely, with all the overhead and complications that
+  it brings)
```

Comment:
The sentence in parenthesis is confusing.

Reply:
Hummm... suggestions?

Reply:
Do we really need this sentence? What about removing it? And avoid references to old code/plugins? I mean: I know that this as part of an RFC document but now we could consolidate this to make it easy for new contributors. 

What do you think?

Reply:
I second @beraldoleal; we don't need to refer to old, removed, code/plugins. Remove the sentence in parenthesis will avoid confusion.

If you prefer to refer it, my suggestion is to go straight to the point, like: "Once, Avocado had remote runner plugins, with all the overhead and complication brought by them."

Reply:
> I second @beraldoleal; we don't need to refer to old, removed, code/plugins. Remove the sentence in parenthesis will avoid confusion.
> 
> If you prefer to refer it, my suggestion is to go straight to the point, like: "Once, Avocado had remote runner plugins, with all the overhead and complication brought by them."

Pushed 4fcfc53856b4e96c3d95fee5d6e186bc7f25d380 with that sentence removed.